### ISSUE TITLE
v1.1.0 on geminipda

### ISSUE THREAD

**Issue by FrankBS** [Huessenbergnetz/Intfuorit#9]
Starting it on geminipda selects tablet MainPage shows empty page and throws a reference error line 53 for 'error'
I changed in harbour-intfuorit.qml the word 'tablet' to 'phone' as workaround and can use it on geminipda. Looks like tablet part is unfinished.

**Comment by FrankBS** [comment]
The error was readable in terminal.

The change was line 22 in harbour-infuorit.qml.

**Comment by buschmann23** [comment]
Yes, you are right. The tablet part is unfinished. As SailfishOS tablet “market“ is even much smaller then the smartphone part, it is not a really high priority for me.
